pkg/odo/cli/service/operator_backend.go

```diff
@@ -261,5 +267,85 @@ func (b *OperatorBackend) buildCRDfromParams(o *CreateOptions, csv olm.ClusterSe
 		return nil, fmt.Errorf("the %q resource doesn't exist in specified %q operator", b.CustomResource, o.ServiceType)
 	}
 
-	return service.BuildCRDFromParams(cr, o.ParametersMap)
+	return svc.BuildCRDFromParams(cr, o.ParametersMap)
+}
+
+func (b *OperatorBackend) DescribeService(o *DescribeOptions, serviceName, app string) error {
+
+	clusterList, _, err := svc.ListOperatorServices(o.KClient)
+	if err != nil {
+		return err
+	}
+	var clusterFound *unstructured.Unstructured
+	for i, clusterInstance := range clusterList {
+		fullName := strings.Join([]string{clusterInstance.GetKind(), clusterInstance.GetName()}, "/")
+		if fullName == serviceName {
+			clusterFound = &clusterList[i]
+			break
+		}
+	}
+
+	devfileList, err := svc.ListDevfileServices(o.EnvSpecificInfo.GetDevfileObj())
+	if err != nil {
+		return err
+	}
+	devfileService, inDevfile := devfileList[serviceName]
+
+	item := NewServiceItem(serviceName)
+	item.InDevfile = inDevfile
+	item.Deployed = clusterFound != nil
+	if item.Deployed {
+		item.Manifest = clusterFound.Object
+	} else if item.InDevfile {
+		item.Manifest = devfileService.Object
+	}
+
+	if log.IsJSON() {
+		machineoutput.OutputSuccess(item)
+		return nil
+	}
+
+	return HumanReadableOutput(os.Stdout, item)
+}
+
+// HumanReadableOutput outputs the list of projects in a human readable format
+func HumanReadableOutput(w io.Writer, item *serviceItem) error {
+	fmt.Fprintf(w, "Version: %s\n", item.Manifest["apiVersion"])
+	fmt.Fprintf(w, "Kind: %s\n", item.Manifest["kind"])
+	metadata, ok := item.Manifest["metadata"].(map[string]interface{})
+	if !ok {
+		return errors.New("unable to get name from manifest")
+	}
+	fmt.Fprintf(w, "Name: %s\n", metadata["name"])
+	spec, ok := item.Manifest["spec"].(map[string]interface{})
+	if !ok {
+		return errors.New("unable to get specifications from manifest")
+	}
+
+	fmt.Fprintln(w, "Parameters:")
```

Comment:
Do you think this should be `Specification` to be more inline with k8s `spec`, or does `Parameters` seem more user-friendly?

Reply:
Same as above, maybe `Parameters` is the way to go.